uktrade/data-hub-frontend
# Assign a Lead ITA to company

## Description of change
Users need to be able to assign an advisers to a company as a Lead ITA. Previously you could only assign yourself, whereas now you can assign other advisers. This feature is currently feature flagged under `manage-lead-adviser` 

## Test instructions
1. Enable feature flag  `manage-lead-adviser`
2. Visit a company that isn't on the One List
3. Select the Lead adviser tab
4. Search for an adviser and submit

_What should I see?_

## Screenshots
### Before
![Screenshot 2020-06-15 at 11 40 33](https://user-images.githubusercontent.com/10154302/84648556-1a55a200-aefd-11ea-93e3-6fa6533e957c.png)


### After
![Screenshot 2020-06-15 at 11 40 15](https://user-images.githubusercontent.com/10154302/84648584-22addd00-aefd-11ea-9bca-ee00292452ba.png)

## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"
